src/App.js

```diff
@@ -32,7 +33,8 @@ class App extends Component {
     fetch(url + this.state.userQuery)
       .then(response => response.json())
       .then(data => this.setState({
-        batch: data.query.search
+        batch: data.query.search,
+        footer: 'footer not-fixed'
```

Comment:
if this state will only be used as a className, maybe name it "footerClass".

e.g. during dev, always think - if I revisit this code in 2 years, what can I do to most easily help myself understand what I did?

(more realistic situation - how do I help the person adding changes to this code (most likely someone other than myself) understand what I did)